libantworld/renderer.h

```diff
@@ -28,7 +28,7 @@ using namespace units::length;
 class Renderer
 {
 public:
-    Renderer(unsigned int cubemapSize = 256, double nearClip = 0.001, double farClip = 1000.0,
+    Renderer(GLsizei cubemapSize = 256, double nearClip = 0.001, double farClip = 1000.0,
```

Comment:
I feel using ``unsigned int`` is a bad habit of mine (and one that Thomas shares so my chances of getting rid of it are slim)